examples/gno.land/r/gov/dao/v2/render.gno

```diff
@@ -0,0 +1,124 @@
+package govdao
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/dao"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/moul/txlink"
+	"gno.land/r/demo/users"
+)
+
+func Render(path string) string {
+	var out bytes.Buffer
+
+	if path == "" {
+		out.WriteString("# GovDAO Proposals\n\n")
+		numProposals := d.Size()
+
+		if numProposals == 0 {
+			out.WriteString("No proposals found :(") // corner case
+			return out.String()
+		}
+
+		offset := uint64(0)
+		if numProposals >= 10 {
+			offset = uint64(numProposals) - 10
+		}
+
+		// Fetch the last 10 proposals
+		proposals := d.Proposals(offset, uint64(10))
+		for i := len(proposals) - 1; i >= 0; i-- {
+			prop := proposals[i]
+
+			title := prop.Title()
+			if len(title) > 40 {
+				title = title[:40] + "..."
+			}
+
+			propID := offset + uint64(i)
+			out.WriteString(ufmt.Sprintf("## [Prop #%d - %s](/r/gov/dao/v2:%d)\n\n", propID, title, propID))
+			out.WriteString(ufmt.Sprintf("#### Status: %s\n\n", strings.ToUpper(prop.Status().String())))
```

Comment:
Why "h4"?

Reply:
please remove h4 here and everyehwere else

Reply:
Replaced it with `****`